unity-renderer/Assets/DCLPlugins/ECS7/Systems/ScenesUiSystem/ECSScenesUiSystem.cs

```diff
@@ -378,6 +387,22 @@ private void SetSceneUiVisibility(int sceneNumber, bool enabled)
                 new StyleEnum<DisplayStyle>(enabled ? DisplayStyle.Flex : DisplayStyle.None);
         }
 
+        private void UpdatePortableExperiencesVisibilityByFeatureToggle(IParcelScene currentScene, IEnumerable<IParcelScene> scenes)
```

Comment:
```suggestion
        private void UpdatePortableExperiencesVisibilityByFeatureToggle(IParcelScene currentScene, IReadOnlyList<IParcelScene> scenes)
```
I would prefer to avoid any `IEnumerable` or `foreach` iteration if it can be avoided and in this scenario it looks like we could use `IReadOnlyList`and the good old `for (..)`

Reply:
`foreach` operations doesnt generate any GC nowdays thus the performance gain would be imperceptible. Anyway the change has been applied